src/API.ts

```diff
@@ -93,5 +92,7 @@ export async function apiRequest(
     instance.interceptors.response.use(_handleResponse, _handleError);
   }
 
-  return instance.request(axiosConfigAdapter(method, path, options));
+  return instance.request(
+    axiosConfigAdapter(method, stripExtraSlash(path), options),
```

Comment:
Are you sure we need this? In Volto we never had to do this. We can assume the paths will be correctly formed (eg. not ending in slash. In which moment the extra trailing slash is present?

Reply:
It was not necessary, I introduced it for the sake of maintaining clarity. I have removed it as per your suggestion.